src/main/scala/com/monsanto/arch/cloudformation/model/resource/CloudFormation.scala

```diff
@@ -38,3 +38,23 @@ object `AWS::CloudFormation::WaitCondition` {
   implicit val format: JsonFormat[`AWS::CloudFormation::WaitCondition`] = jsonFormat6(`AWS::CloudFormation::WaitCondition`.apply)
 }
 
+case class `AWS::CloudFormation::Stack`(name: String,
+                                        TemplateURL: Token[String],
+                                        TimeoutInMinutes: Option[Int] = None,
```

Comment:
Looking at the documentation at: http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-stack.html it mentions the type should be a String. Do we need this to be a StringBackedInt?